CHANGELOG.rst

```diff
@@ -1,5 +1,13 @@
-OTP reading library change log
-==============================
+lib_otpinfo change log
+======================
+
+2.2.0
+-----
+
+  * ADDED: Support for xs3 architecture
+  * CHANGED: Use the tools otp library for reading from the OTP memory
```

Comment:
might be worth being more specific about what "tools" is. "XTC tools" maybe?

Reply:
done